File: plane_seat.py
def plane_seat(a):
    lstA = list(a)
    if len(lstA)>2:
        num = int(''.join(lstA[0:2]))
        spot = str(lstA[2])
    else:
        num = int(''.join(lstA[0]))
        spot = str(lstA[1])
    left = ['A','B', 'C']
    middle = ['D', 'E', 'F']
    right = ['G', 'H', 'K']
    
    if num <= 20:
        if spot in left:
            return 'Front-Left'
        elif spot in middle:
            return 'Front-Middle'
        elif spot in right:
            return 'Front-Right'
    elif num <=40:
        if spot in left:
            return 'Middle-Left'
        elif spot in middle:
            return 'Middle-Middle'
        elif spot in right:
            return 'Middle-Right'
    elif num <= 60:
        if spot in left:
            return 'Back-Left'
        elif spot in middle:
            return 'Back-Middle'
        elif spot in right:
            return 'Back-Right'
    return 'No Seat!!'

File: planeSeat.js
function planeSeat(a){
    let num, spot;
     if (a.length>2){
      num = Number(a.slice(0,2));
       spot = a.slice(2);
     }else{
       num = Number(a.slice(0,1));
       spot = a.slice(1);
     }
    let left = ['A', 'B', 'C'];
    let mid = ['D', 'E', 'F'];
    let right = ['G', 'H', 'K'];
   if(num<=20){
     if (left.includes(spot)){
       return 'Front-Left';
     }else if (mid.includes(spot)){
       return 'Front-Middle';
     }else if (right.includes(spot)){
       return 'Front-Right';
     }
   }else if (num <=40){
     if (left.includes(spot)){
       return 'Middle-Left';
     }else if (mid.includes(spot)){
       return 'Middle-Middle';
     }else if (right.includes(spot)){
       return 'Middle-Right';
     }
   }else if (num <= 60){
      if (left.includes(spot)){
       return 'Back-Left';
     }else if (mid.includes(spot)){
       return 'Back-Middle';
     }else if (right.includes(spot)){
       return 'Back-Right';
     }
   }
    return 'No Seat!!';
  }